src/util/lazyWithRetry.tsx

```diff
@@ -0,0 +1,62 @@
+import { lazy } from "react";
+import { delay } from "./helpers";
+
+// This function is a wrapper around React.lazy that will retry the import
+// keeping the function type signature the same as React.lazy
+// using "any" type here allows for components with any type interface to be imported
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const lazyWithRetry = <T extends React.ComponentType<any>>(
+  importFunction: () => Promise<{ default: T }>,
+  retries = 3,
+  delayTime = 1000,
```

Comment:
```suggestion
  delayTime = 250,
```
I don't think we should set it this high.
Also I don't think the delay time and the retires should be configurable. We can just define constants for it.

Reply:
thanks for the catch, the delay time was set a little high so it was easy to test if the page would load again if I turn on the network. I've made the above changes now :)